packages/commonwealth/server/routes/updateChain.ts

```diff
@@ -133,8 +135,12 @@ const updateChain = async (
   if (custom_stages) chain.custom_stages = custom_stages;
   if (terms) chain.terms = terms;
   if (snapshot) chain.snapshot = snapshot;
-  if (chat_enabled) chain.chat_enabled = chat_enabled;
+  // Set default allow/deny permissions
+  chain.default_allow_permissions = default_allow_permissions;
+  chain.default_deny_permissions = default_deny_permissions;
 
+  chain.default_allow_permissions = default_allow_permissions || BigInt(0);
```

Comment:
Why do we set them twice and do a meaningless `|| BigInt(0)`?

Reply:
the || BigInt(0) is to guarantee that there is a default value set if default_allow_permissions is null. The previous lines are useless though I just removed them